cmds/core/tee/tee_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestTee(t *testing.T) {
+	tempDir := t.TempDir()
+
+	tests := []struct {
+		input  string
+		args   []string
+		append bool
+	}{
+		{
+			"hello",
+			[]string{filepath.Join(tempDir, "a1"), filepath.Join(tempDir, "a2")},
+			false,
+		},
+		{
+			"a\nb\n",
+			[]string{filepath.Join(tempDir, "b1")},
+			true,
```

Comment:
Does this actually test the append functionality?    Same for the first test case - it doesn't actually verify that 'overwrite' happens instead of 'append' right? Can the test cases/code be rewritten to ensure that `append` works properly (and doesn't work when it's not enabled)? Thanks!

Reply:
yep, make sense to test not only if oflags is legit, but if actually appends something.

Reply:
Thanks!